### PR TITLE
YP popup translations

### PR DESCRIPTION
**Pull request by LettyYeti**
Add human-approved translations (with exception of Polish) and adjust popup layout to accommodate all translations. 

Tested on PC & full visual check against localisation spreadie

### CODE REVIEW

File: src/asmcnc/core_UI/job_go/popups/popup_yetipilot_settings.py [hunk 1]
```diff
@@ -128,8 +128,8 @@ def __init__(self, screen_manager, localization, machine, database, yetipilot, v
         sum_of_middle_heights = subtitle_height  + radio_BL_height + body_BL_height
         close_button_BL_height = vertical_BL_height - sum_of_middle_heights
 
-        dropdowns_container_width = 350
-        dropdowns_width = 270
+        dropdowns_container_width = 350 - 20
```
Comment (DennisKiselev) [inline src/asmcnc/core_UI/job_go/popups/popup_yetipilot_settings.py:131]: Is this intentional? In place of just 330

Comment (LettyYeti) [reply src/asmcnc/core_UI/job_go/popups/popup_yetipilot_settings.py]: Ah it was so I could keep track of the changes I was making to the background at the same time 😂. Have updated, let me know if you're happy @DennisKiselev 😁

**Comment by LettyYeti** [comment]
image
image
image
image

**Comment by LettyYeti** [comment]
Have also changed colour and weight of YetiPilot.
Have left in commented out code in case someone asks me to change it back 😂

**Pull request merged**